python/questions_test/chapter1_test.py

```diff
@@ -0,0 +1,35 @@
+
+import pytest
+from questions import chapter1
+
+
+
+def test_chapter_1_question_1():
```

Comment:
Given that the file name is `chapter1_test`, it's redundant to include it in the test method name. `test_question1` is a better method name.